echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/CDEventsNotificationAgent.java

```diff
@@ -70,16 +71,22 @@ public void sendNotifications(
             preference, application, event, config, status, getSpinnakerUrl());
     log.info(
         "Sending CDEvent {} notification to events broker url {}", cdEventsType, eventsBrokerUrl);
-    Response response = cdEventsSenderService.sendCDEvent(cdEvent, eventsBrokerUrl);
+    Response<ResponseBody> response = cdEventsSenderService.sendCDEvent(cdEvent, eventsBrokerUrl);
     if (response != null) {
-      log.info(
-          "Received response from events broker : {} {} for execution id {}. {}",
-          response.getStatus(),
-          response.getReason(),
-          executionId,
-          response.getBody() != null
-              ? new String(((TypedByteArray) response.getBody()).getBytes())
-              : "");
+      try {
+        log.info(
+            "Received response from events broker : {} {} for execution id {}. {}",
+            response.code(),
+            response.message(),
+            executionId,
+            response.body() != null ? response.body().string() : "");
+      } catch (IOException e) {
+        log.info(
+            "Received response from events broker : {} {} for execution id {}",
```

Comment:
Include some indication that there was an error serializing the response body / some differentiation from the above message?